sdk/core/azure-core/CHANGELOG.md

```diff
@@ -14,6 +14,7 @@
 ### Other Changes
 
 - [[#4756]] (https://github.com/Azure/azure-sdk-for-cpp/issues/4756) `BearerTokenAuthenticationPolicy` now uses shared mutex lock for read operations.
+- Added internal provisions for extended `Azure::DateTime` parsing.
```

Comment:
Detect TZ offset in Identity, and append "+/-HH:MM" to the datetime that az cli returns.

Reply:
Done.